.gitattributes

```diff
@@ -1,3 +1,4 @@
 *.ipynb filter=nbstripout
 *.ipynb diff=ipynb
 stglib/_version.py export-subst
+mc3575.asc filter=lfs diff=lfs merge=lfs -text
```

Comment:
Looks good, just need to remove this line from this .gitattibutes since you added it to the one in the tests directory.